### ISSUE TITLE
[Typo in example code]📜

### ISSUE THREAD

**Issue by juliabolt** [hedyorg/hedy#5048]
**Describe the issue**
What level, language and adventure does this concern?

Level 5, every language, adventure Draw it!

The code that is given to copy and complete further, misses a quotation mark, which, if not fixed, gives weird errors saying that `chosen_letter` is a list.

```
chosen_letter is ask 'Which letter would you like to see? F, E or L?
if _
```

**Expected text**
What should we write instead?

A quotation mark should be added at the end of the first line of code:

```
chosen_letter is ask 'Which letter would you like to see? F, E or L?'
if _
```

**Comment by Felienne** [comment]
Hi @juliabolt! Thanks fort letting us know. Any chance you can pick this up? If you want to, I can show you how to do it!

**Comment by juliabolt** [comment]
Yes maybe you can show me this week during the Hedy Hackathon for example!

**Comment by juliabolt** [comment]
I found this in the yaml file with all the adventures, so I can adjust this in the English and Dutch yaml file?

**Comment by Felienne** [comment]
Yes! And if you want to do it really wel... you can also see if the English text is still present in other languages (I can also show you tomorrow)
